catalog/networking/network/vpn.yaml

```diff
@@ -11,22 +11,35 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 apiVersion: compute.cnrm.cloud.google.com/v1beta1
-kind: ComputeTargetVPNGateway
+kind: ComputeVPNGateway
 metadata:
-  name: network-name-vpn-gateway # kpt-set: ${network-name}-vpn-gateway
+  name: network-name-ha-vpn-gateway # kpt-set: ${network-name}-ha-vpn-gateway
   namespace: networking # kpt-set: ${namespace}
   annotations:
     cnrm.cloud.google.com/project-id: project-id # kpt-set: ${project-id}
     cnrm.cloud.google.com/blueprint: cnrm/landing-zone:networking/v0.3.0
 spec:
-  description: "Compute VPN Gateway"
+  description: "Compute HA VPN Gateway"
   networkRef:
     name: network-name # kpt-set: ${network-name}
   region: us-central1 # kpt-set: ${region}
 ---
 apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeExternalVPNGateway
+metadata:
+  name: network-name-ext-vpn-gateway # kpt-set: ${network-name}-ext-vpn-gateway
+  namespace: networking # kpt-set: ${namespace}
+  annotations:
+    cnrm.cloud.google.com/project-id: project-id # kpt-set: ${project-id}
+    cnrm.cloud.google.com/blueprint: cnrm/landing-zone:networking/v0.3.0
+spec:
+  redundancyType: "SINGLE_IP_INTERNALLY_REDUNDANT"
+  interface:
+    - id: 0
+      ipAddress: "8.8.8.8" # kpt-set: ${vpn-tunnel-peerIp}
+---
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
 kind: ComputeVPNTunnel
```

Comment:
Shouldn't we be creating two tunnels (assuming https://cloud.google.com/network-connectivity/docs/vpn/concepts/topologies#1-peer-1-address is what you were configuring)

Reply:
Valid point. I've configured the peer gateway to use "TWO_IPS_REDUNDANCY" instead and configured another tunnel so they both have an interface to hook on to.